nexus/reconfigurator/planning/src/planner.rs

```diff
@@ -563,121 +557,49 @@ impl<'a> Planner<'a> {
                         "wanted_to_place" => num_zones_to_add,
                     );
 
-                    // Adjust `num_zones_to_add` downward so it's consistent
-                    // with the number of zones we're actually adding.
-                    num_zones_to_add = i;
-
                     break;
                 }
-            }
-        }
-
-        // For each sled we need to change, actually do so.
-        let mut new_zones_added = 0;
-        for (sled_id, additional_zone_count) in sleds_to_change {
-            // TODO-cleanup This is awkward: the builder wants to know how many
-            // total zones go on a given sled, but we have a count of how many
-            // we want to add. Construct a new target count. Maybe the builder
-            // should provide a different interface here?
```

Comment:
This was my very least favorite question in the planner. Thanks for answering it!